docs/bnb_smart_chain/validator/run-val.md

```diff
@@ -0,0 +1,123 @@
+
+# Run BSC Validator
+
+## Validator Hardware Requirements
+
+### Mainnet
+- VPS running recent versions of Mac OS X or Linux.
+- **IMPORTANT** 3T GB of free disk space, solid-state drive(SSD), gp3, 8k IOPS, 500 MB/S throughput, read latency <1ms (if start with snap sync, it will need NVMe SSD).
+- 16 cores of CPU and 64 gigabytes of memory (RAM).
+- Suggest m5zn.3xlarge or r7iz.4xlarge instance type on AWS, or c2-standard-16 on Google cloud.
+- A broadband Internet connection with upload/download speeds of 5 megabyte per second
+
+### Testnet
+- CPU: 4core, Intel(R) Xeon(R) Platinum 8175M CPU @ 2.50GHz
+- Memory: 16 GB
+- Disk: ~1.5 TB
+- Network Bandwidth: Upto 25 Gbps
+- EBS Bandwidth: Upto 9.5 Gbps
+
+## Setup Validator Node
+
+### 1. Install BSC Fullnode
+
+Follow the instructions [here to set up a full node](fullnode.md).
```

Comment:
This is an invalid link

Reply:
Yes, but the fullnode runbook is not in this scope, should fix it after adding the fullnode part.

Reply:
fixed